### PR TITLE
Validation script

### LINKED ISSUE #107: Write validation scripts for data before pushing to s3
## Geoid files
- [x] Validate total number of geoid files is expected: county number + state number

- [x] Validate first key in geoid file (06085.json) is run_date. No period, semicolon, colon, etc in key.

- [x] Validate next key can only be low, med or high

- [x] Validate object includes name, max, over, r0, and vals

- [x] Validate length of vals is same as overall length of dates

- [x] Validate indicators only includes expected incidI, incidH, etc from outcomes.json

## statsForMap 

- [x] Validate state key is first, and then county key

- [x] Validate scenario_name matches geoid scenario_name

- [x] Keys are exactly as expected - from outcomes.json

# countyBoundaries

- [x] Check stats exists in statsForMap and countyBoundaries

## Actuals files

- [x] Validate number of expected actuals files

- [x] Contains expected reported data 

## Requested Feature

- [x] Run and test validator on just one geoid

### PR DESCRIPTION
**Pull request by lxu213**
Fixes #107

### CODE REVIEW

**Comment by lxu213** [comment]
@veev can you review this and let me know if there's any other validation I missed?

File: src/scripts/validate.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+    if len(counties) < len(expected_counties):
+        raise Exception("Expecting at least {} county-level actual.json files, not {}"
+            .format(len(expected_counties), len(counties)))
+
+
+def validate_reported_indicators(actual_obj: dict):
+    ''' 
+    Validate actual file includes expected reported indicators 
+    Confirm date format and value type are '%Y-%m%d' and int, respectively
```
Comment (veev) [inline src/scripts/validate.py:161]: should there be a "-" between %m and %d?

Comment (lxu213) [reply src/scripts/validate.py]: Yes, thanks

**Review by veev — COMMENTED** [review]
@lxu213 this looks pretty thorough. wasn't sure if we're capturing the issue that broke the most recent file josh made - essentially checking whether all the keys are included in the files.

**Comment by lxu213** [comment]
@veev Yes! we are checking for all keys.

**Pull request merged**